locales/notes.ca.properties

```diff
@@ -0,0 +1,80 @@
+# General note/notebook operations
+notebooks=Blocs de Notes
+tap-anywhere-to-start=Premeu a qualsevol lloc per iniciar
+note-info=Informacio de la nota
+add-to-contacts=Afegir a Contactes
+new-notebook=Crear Bloc de Notes
+notebook-all=Totes les notes
+notebook-trash=Paperera
+notebook-action-title=Editar Bloc de notes
+notebook-action-rename=Canviar el nom
+notebook-action-delete=Eliminar
+prompt-rename-notebook=Canviar el nom del Bloc de notes:
+prompt-delete-notebook=Premeu OK per eliminar aquest Bloc de notes. \n\nObservaci\u00f3: totes les vostres notes seran enviades a la paperera.
```

Comment:
Hmm, not quite. I mean basically copy and paste the literal string "Observació" in there without any encoding. The app will pick it up correctly. 

Here's a screen shot of the file in my text editor: 

![notes ca prop](https://cloud.githubusercontent.com/assets/2423092/12369145/f64b6bd0-bba7-11e5-903b-bef864dff923.png)